hub/features/class_label.py

```diff
@@ -1,26 +1,93 @@
 from typing import List
 
-import hub.features.feature_connector as feature_connector
+from hub.features.features import Tensor
 
-FeatureConnector = feature_connector.FeatureConnector
 
+def _load_names_from_file(names_filepath):
+  with open(names_filepath, "r") as f:
+    return [
+        name.strip()
+        for name in f.read().split("\n")
+        if name.strip()  
+    ]
 
-class ClassLabel(FeatureConnector):
-    def __init__(
-        self, num_classes: int = None, names: List[str] = None, names_file: str = None
-    ):
-        pass
+
+class ClassLabel(Tensor):
+    def __init__(self, num_classes: int = None,
```

Comment:
As I remember TFDS allows to input names and names_file in num_classes stead. So in future we would like first argument which happends to be num_classes support all 3:

- int
- List of str
- filepath (str) where names are stored.